doc/SIDEBAR_MOD.md

```diff
@@ -475,63 +487,123 @@ mapped as closely as possible to the spectrum of colors, with one exception - va
 "normal" value or range always use white (`c_white`) when the value is within normal.
 
 
-## `phrases`
+## `flags`
+
+Widgets can use flags to specify special behaviors:
+
+```json
+{
+  "id": "my_widget",
+  "type": "widget",
+  "style": "text",
+  "label": "My Widget",
+  "var": "my_widget_var",
+  "flags": [ "W_LABEL_NONE", "W_DISABLED" ]
+}
+```
+
+Here are some flags that can be included:
 
-Some widgets can take advantage of "phrases" - definitions for what text/values to display and
+| Flag id            | Description
+|---                 |---
+| `W_LABEL_NONE`     | Prevents the widget's label from being displayed in the sidebar
+| `W_DISABLED`       | Makes this widget disabled by default (only applies to top-level widgets/layouts)
+| `W_DYNAMIC_HEIGHT` | Allows certain multi-line widgets to dynamically adjust their height
+
+
+# Phrases and conditions
+
+Widgets can take advantage of "phrases" - definitions for what text/values to display and
```

Comment:
```suggestion
Widgets can take advantage of "clauses" - definitions for what text/values to display and
```

I definitely like the word "clauses" for this. It's a common word with a [well-understood meaning](https://www.lexico.com/grammar/clauses) in the grammatical sense that aligns well with what this feature is doing. Plus, *nothing* else in our code base is using this word yet, so text search will be our ally.